sonar-scanner-engine/src/main/java/org/sonar/scanner/scan/ProjectScanContainer.java

```diff
@@ -271,7 +272,11 @@ protected void doAfterStart() {
     }
   }
 
-  private static String toDisplayName(BranchType branchType) {
+  private String pullRequestBaseToDisplayName(@Nullable String pullRequestBase) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make "pullRequestBaseToDisplayName" a "static" method. ([squid:S2325](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2325))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fjanos%2FSONAR-10365%2Fmigrate-data&issues=AWF65MK3GUleEzPZFu4E&open=AWF65MK3GUleEzPZFu4E)

[ISSUE_KEY]: AWF65MK3GUleEzPZFu4E